src/py42/_internal/clients/pds.py

```diff
@@ -0,0 +1,24 @@
+from py42.clients import BaseClient
+
+
+class PreservationDataServiceClient(BaseClient):
+    def find_file_versions(self, file_md5, file_sha256, device_id, paths):
```

Comment:
It lets you pass both md5 and sha256?  Should we just make two methods here, `find_file_versions_by_sha256()` and `find_file_versions_by_md5()` ?

Reply:
I guess since this is an internal client, it isn't that important